### ISSUE TITLE
Syncdb throws Elasticsearch Error

### ISSUE THREAD

**Issue by dorvak** [amcat/amcat#407]
Using the elasticsearch ICU-plugin (v 2.4.2)  I get the following erro while running the syncb-script:

    File "/home/admin/env/amcat-env/lib/python2.7/site-packages/elasticsearch/connection/base.py", line 97, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
    elasticsearch.exceptions.RequestError: TransportError(400, u'IndexCreationException[[amcat] failed to create index]; nested: IllegalArgumentException[Custom Analyzer [default] failed to find filter under name [icu_folding]]; ')

I upgraded the ICU plugin to the current version (2.7.0), this leads to another error:

    File "/home/admin/env/amcat-env/lib/python2.7/site-packages/elasticsearch/connection/base.py", line 97, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
    elasticsearch.exceptions.RequestError: TransportError(400, u'MapperParsingException[mapping [article]]; nested: ElasticsearchIllegalArgumentException[failed to find postings_format [bloom_default]]; ')

System info: 
- CentOS 6.7
- Postgres 8.4.20
- Lucene 4.10.4 
- Elasticsearch 1.7.2

BTW: Might docker be an option for you to ship the app?

**Comment by vanatteveldt** [comment]
Are you using the master branch? I know that there was a change in which mapping were supported. I'm busy teaching today, tomorrow I will be online on google chat and we can see what our current configuration is.

Docker might be a good idea. We considered shipping VM's earlier but thought it was overkill, maybe docker is the right tool. Do you have experience with it?

**Comment by dorvak** [comment]
Yes, I cloned from the github repo. Don't worry, it's not time-critical (we might use Amcat for our manual content analysis and I will probably extend it (although I#m not firm with Django, always used flask).

Haven't worked with docker yet, but i think it's quite lightweight for a VM (and should run fairly efficient). Plus: users might run Amcat on Heroku (etc.)

**Comment by vanatteveldt** [comment]
Hey Till,

I looked into it because I ran into the same problems in a new install. Your problem was "caused" by using a new version of elastic, I think if you used the elastic in the install instructions it would have worked (because that's what our unit tests do).

I think elastic disabled bloom by default in 1.4 and removed them sometime later, they say that the optimizations were incorporated in the standard index. So the line with bloom_default can just be dropped from settings/elastic.py

Let me know when you want to extend the manual coding interface, it might be a good idea to have a chat session (probably also involvind @martijnbastiaan  and/or @CJStuart who know the javascript part much better than I do)
